README.md

```diff
@@ -184,6 +197,31 @@ the interface numbers will (probably) be different. The interface number is
 arbitrary, so if it is already assigned replace it with a number of your
 choice.
 
+Example NixOS output:
+
+    networking.wireguard.interfaces = {
```

Comment:
I've had a thought -- is this some kind of standard serialized output? If so we might be better off encoding it directly rather than using a template.